.github/workflows/test.yaml

```diff
@@ -159,9 +159,11 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: install with geo
-        run: pip install -v --prefer-binary -e '.[tests, examples-tests, geo, hvdev, hvdev-geo, dev-extras]'
-      - name: pip list
-        run: pip list
+        run: python -m pip install -v --prefer-binary -e '.[tests, examples-tests, geo, hvdev, hvdev-geo, dev-extras]'
```

Comment:
Good practice to call `python -m pip`, see https://pip.pypa.io/en/stable/cli/pip_install/.